src/dao/pages/CreateADAO/forms/TokenDAOReviewForm.tsx

```diff
@@ -35,6 +37,7 @@ export function TokenDAOReviewForm() {
           }}
         />,
         <FormInput<"governance.token.symbol">
+          key="governance.token.symbol"
```

Comment:
great, this will resolve some of the lint issue's.

Reply:
Yes.